src/components/EventDetails.js

```diff
@@ -16,7 +16,17 @@ const EventDetails = () => {
   const { slug } = useParams();
   const eventData = state.events;
 
-  const event = eventData.find(event => event.path === slug);
+
+  let event, prevEvent, nextEvent;
+
+  eventData.forEach((e, i) => {
+    if (e.path === slug) {
+      event = e;
+      prevEvent = eventData[i-1];
```

Comment:
I think if we end up making the events user-sortable, this would still work nicely 👍 👍